CONTRIBUTION.md

```diff
@@ -0,0 +1,82 @@
+### Recommended development flow
+
+The most natural way to develop che-machine-exec it's using Eclipse CHE workspace.
+To install Eclipse CHE you need to choose infrastructure(openshift, kubernetes or docker) and set up it.
```

Comment:
would be nice to have link to docs how to setup environment

Reply:
Done.